[DOC]: Update Langchain integration example and fix broken link

ishaksebsib opened the pull request:
## Description of changes

 - Improvements & Bug fixes
	 - Removed the code example from the Langchain integration documentation as per @jeffchuber's request.
	
 - New functionality
	 - n/a

## Test plan
*How are these changes tested?*

- [x] Tests pass locally with `pytest` for python, `yarn test` for js, `cargo test` for rust

## Documentation Changes
*Are all docstrings for user-facing APIs updated if required? Do we need to make documentation changes in the [docs repository](https://github.com/chroma-core/docs)?*

- The example in the docs has been updated. There are no other documentation changes required.

jeffchuber commented:
@ishaksebsib thanks for this! we are going to do a wholesale rewrite of the langchain docs this week. happy to still give you credit on the PR for finding this stuff! can we cut the code example entirely for now?

ishaksebsib commented:
> @ishaksebsib thanks for this! we are going to do a wholesale rewrite of the langchain docs this week. happy to still give you credit on the PR for finding this stuff! can we cut the code example entirely for now?

Hi @jeffchuber,

Thanks for the feedback. I've removed the code example as requested and updated the PR.

Let me know if there's anything else I can assist with. I'm also happy to help with the upcoming documentation rewrite if needed.
